### PR TITLE
Fixes #6979

### LINKED ISSUE #6979: Opening the aspect dropdown and clicking anywhere closes the publisher on FF dev
I'm not sure what's going on but opening the aspect dropdown in the last Firefox developer edition (50.0a2 (2016-08-13) here) inserts a div that take the whole page and triggers the closing of the publisher when clicking anywhere in the page except the dropdown menu (this means when clicking on the dropdown button itself too).

dropdown-backdrop

According to the Bootstrap's docs:

> On mobile devices, opening a dropdown adds a `.dropdown-backdrop` as a tap area for closing dropdown menus when tapping outside the menu, a requirement for proper iOS support.

This is browser specific and do not reproduce on Firefox 48 but maybe it is worth neutralizing this `.dropdown-backdrop` element?

### PR DESCRIPTION
**Pull request by ghost**
Fixes #6979

### CODE REVIEW

File: spec/javascripts/app/views/publisher_view_spec.js [hunk 1]
```diff
@@ -101,6 +101,34 @@ describe("app.views.Publisher", function() {
...
         expect(this.view.$el.find(".poll-creator-container")).toBeVisible();
       });
+
+      it("should close the publisher when clicking outside", function() {
+        expect("#publisher").not.toHaveClass("closed");
+        $("body").click();
+        expect("#publisher").toHaveClass("closed");
+      });
+
+      it("should close the publisher when clicking inside", function() {
```
Comment (svbergerem) [inline spec/javascripts/app/views/publisher_view_spec.js]: "should **not** close..."

File: spec/javascripts/app/views/publisher_view_spec.js [hunk 2]
```diff
@@ -101,6 +101,34 @@ describe("app.views.Publisher", function() {
...
+        expect("#publisher").not.toHaveClass("closed");
+        $("#publisher").find(".publisher-textarea-wrapper").click();
+        expect("#publisher").not.toHaveClass("closed");
+        $("#publisher").find(".aspect_dropdown button").click();
+        expect("#publisher").not.toHaveClass("closed");
+      });
+
+      it("should close the publisher when clicking inside on a mobile", function() {
```
Comment (svbergerem) [inline spec/javascripts/app/views/publisher_view_spec.js]: "should **not** close..."

Comment (AugierLe42e) [inline spec/javascripts/app/views/publisher_view_spec.js]: Right, thanks ;)

**Comment by svbergerem** [comment]
Thanks!

**Pull request merged**